src/components/TransactionTable/TransactionTable.js

```diff
@@ -0,0 +1,214 @@
+import React, { PureComponent } from 'react';
```

Comment:
If we're thinking that we're going to be going down a path with a bunch of different sub-types of tables (and possibly other components) would it be worth "bundling" this in the same directory as the main `Table`? Just trying to think of long term maintainability and what would be the easiest to grok for newbies. Don't have a strong opinion necessarily though.

Reply:
I could see good reasoning either way and am not particularly opinionated on it

Reply:
With most of the logic specific to this table now being removed, I feel like this option (moving it in with Table) would make even more sense. I guess we should think about how someone might be using this and what the most natural way to import it would be too.